src/plugins/plot/src/plot/MCTPlotController.js

```diff
@@ -413,6 +413,21 @@ define([
             return;
         }
 
+        const isPinchToZoom = event.ctrlKey === true;
+        let isZoomIn = event.wheelDelta < 0;
+        let isZoomOut = event.wheelDelta >= 0;
+
+        //Flip the zoom direction if this is pinch to zoom
+        if (isPinchToZoom) {
+            if (isZoomIn === true) {
+                isZoomOut = true;
+                isZoomIn = false;
+            } else if (isZoomOut === true) {
+                isZoomIn = true;
+                isZoomOut = false;
+            }
+        }
```

Comment:
can we use ES6 de-structuring to make this logic simpler? ie
`if (isPinchZoom) {
[isZoomIn, isZoomOut] = [isZoomOut, isZoomIn]
}` ?

Reply:
I intentionally didn't do this because Andrew had mentioned not to do this if possible :)